src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/PVTFunctionBase.hpp

```diff
@@ -112,12 +114,23 @@ class PVTFunctionBase
 
   virtual ~PVTFunctionBase() = default;
 
+
+  /// Struct containing output options
+  struct TableOutputOptions
+  {
+    /// Output PVT in CSV file
+    bool writeCSV;
+    /// Output PVT in log
+    bool writeInLog;
+  };
+
+
   using CatalogInterface = dataRepository::CatalogInterface< PVTFunctionBase,
                                                              string const &,
                                                              array1d< string > const &,
                                                              array1d< string > const &,
                                                              array1d< real64 > const &,
-                                                             bool const >;
+                                                             TableOutputOptions >;
```

Comment:
```suggestion
                                                             TableOutputOptions const & >;
```
It's a big change that needs to trickle everywhere but should make it clear it's input only.

Reply:
I agree with the `const ` but it's lighter to pass by copy. 
If in the case of `catalog`, all variable must by passed by `const ref` I'll do the same

Reply:
What do you think @dkachuma?

Reply:
What do you think @dkachuma?